src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Serialization/MessagePack/SerializationFormat.cs

```diff
@@ -9,5 +9,5 @@ internal static class SerializationFormat
     // or any of the types that compose it changes. This includes: RazorConfiguration,
     // ProjectWorkspaceState, TagHelperDescriptor, and DocumentSnapshotHandle.
     // NOTE: If this version is changed, a coordinated insertion is required between Roslyn and Razor for the C# extension.
-    public const int Version = 3;
+    public const int Version = 4;
```

Comment:
Given you're backing out the serialization change, this feels unnecessary. Are you just trying to get to a "clean slate" because v3 is a little muddy now?

Reply:
Yeah, this probably isn't necessary. Essentially, your change here just rolls it back to what version 3 was before the recent change to `RazorConfiguration` serialization. Since the version didn't get updated then, I can't see updating it now.